notes.md

```diff
@@ -300,6 +327,23 @@ Overlay tracks **SHOULD** be rendered in the same channel as the track its linke
 When content is found in such a track, it **SHOULD** be played on the rendering channel
 instead of the original track.
 
+## Track Links
+
+A track link indicates that a given subtitle track is suitable for use with a given audio track,
+and that it **SHOULD** be eligible for autoselection when that audio track is selected.
+
+For instance:
+- A forced subtitle track containing onscreen text consistent with the dialogue in an audio track
+  of the same language should be linked to that track, but should not be linked to another audio track
+  featuring a different translation into the same language.
+- A subtitle track featuring a transcription of an audio commentary track should be linked to
+  that audio commentary track, but not to other tracks featuring separate commentary recordings.
+
+Track links are only necessary when the set of tracks available creates ambiguity about which
+subtitles are appropriate for which audio, and function as a hint to the player to improve
+its automatic track selection behavior. In cases where there is no such ambiguity, there is
+no need to explicitly specify track links.
```

Comment:
A "default value" for `TrackLinks` (the list of `TrackLink` for a track) would help. An empty `TrackLink` means the track is not associated with any other track. So if you remux a file and remove an audio track, you also remove all the TrackLink for that track. The `TrackLinks` for those tracks might end up empty.

So setting a `TrackLinks` to empty is the opposite of having no `TrackLinks` at all which means it applies to all audio/video tracks.

Reply:
(Removed for now)